test/strtest.jl

```diff
@@ -7,14 +7,20 @@ const pkglist =
      "Format", "StringLiterals", "Strs", "StrICU"]
 
 const mparse = @static VERSION < v"0.7.0-DEV" ? parse : Meta.parse
-_rep(str, a, b) = @static VERSION < v"0.7.0-DEV" ? replace(str, a, b) : replace(str, a => b)
+@static if VERSION < v"0.7.0-DEV"
```

Comment:
Did you make any changes to `strtest.jl`?  If not, remove it from your PR, so you don't have conflicts.

Reply:
Not that I know of